### ISSUE TITLE
Contacts

### ISSUE THREAD

**Issue by DavidValin** [nash-md/meow#5]
Hello! Very great software! I just installed it and just as it is seems promising!

However, I find that when selling, the contact is very important. We have main contacts in an Account and then the contact for the opportunity. I find that we need to find a way to find communication (history) for a specific contact easy, so that would be "a contact belongs to an account and a contact has many opportunities".

Many times, when opening an opportunity for an account I would like to find other opportunities for these contact. Or given a contact that I just stepped into the sidewalk, be able to quickly find all the previous opportunities we had...

I think the contact deserves a separate table with some fields (role, email, phone, ...) and some views to handle those usecases, if others find them important. What are your thoughts?

**Comment by nash-md** [comment]
@DavidValin not sure why I never got any notifications on the issue you raised. Let me look into this

**Comment by saashqdev** [comment]
Yes, contacts would be a great addition. @nash-md  I'm up for helping add contacts.
